### DISCUSSION TITLE
ADetailer do not swap to separate checkpoint.

### DISCUSSION THREAD

**Discussion by Kmolpat** [lllyasviel/stable-diffusion-webui-forge#976] [General]
It just doesn't work anymore like it used to.

**Comment by ghost** [comment]
Thanks to point out this issue, you're not alone ;-)
It was a very important feature of my personal workflow so I hope devs will find a fix soon...

**Comment by vihangasa14** [comment]
Same here

**Comment by Stektpotet** [comment]
Seems to be a sd-forge issue: 
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/964#discussioncomment-10316697 & https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1026

**Reply by Kmolpat** [reply]
yes, it is. 
When it starts to use ADtailer, it should load the model first, then starts to detail the face. But after the update the model loading behavior, when it starts the detailer, it starts by go to read the prompt with LoRA init and then generate the detail before change the model.

**Comment by ainomad1668** [comment]
I encountered the same issue. 
It improved and cleaned up the face, but it didn’t utilize the separate checkpoint. 

I hope it gets fixed soon.

**Comment by CCpt5** [comment]
Edit: This feature no longer works in Uddetailer either - The developer of that repo commented (see next message)


_____
udetailer is a similar fork (like adetailer). Have you tried that?

The developer is pretty active (very helpful) so if you have tried it and had issues w/ the latest forge maybe mention them in this thread on their repo. They tested w/ Forge two weeks ago and said it works fine: https://github.com/wkpark/uddetailer/issues/96

**Comment by CCpt5** [comment]
Ok, so I made an issue on the uddetailer repo mentioning that the option to use a different checkpoint while inpainting a face was no longer working in Forge. The developer, wkpark, reviewed the issue and found an override feature for model changing that still exists in A1111 was removed from Forge. Apparently this is a breaking change for that feature and any fix would require input from the developer of Forge @lllyasviel. 


I'm just a layman, but I assume this override feature was likely removed due to all of the updated code that was required to properly handle Flux and its large memory footprint. Here's the link to wkpark's reply on the uddetailer repo: https://github.com/wkpark/uddetailer/issues/99#issuecomment-2351485092

**Reply by driqeks** [reply]
can you open an issue about it here?

**Comment by icsl-Jeon** [comment]
https://github.com/Bing-su/adetailer/issues/716
